data/dslr.csv

```diff
@@ -0,0 +1,123 @@
+Nikon, 0x0412, "Nikon D80", 0, 3872, 2592, 6.09,,
+Nikon, 0x0414, "Nikon D40", 0, 3008, 2000, 7.87,,
+Nikon, 0x0416, "Nikon D2XS", 0, 4288, 2848, 5.52,,
+Nikon, 0x0418, "Nikon D40X", 0, 3872, 2592, 6.09,,
+Nikon, 0x041a, "Nikon D300", 0, 4288, 2848, 5.50,,
+Nikon, 0x041c, "Nikon D3", 0, 4256, 2832, 8.45,,
+Nikon, 0x041e, "Nikon D60", 0, 3872, 2592, 6.09,,
+Nikon, 0x0420, "Nikon D3X", ptp_flag_lv, 6048, 4032, 5.95,,
+Nikon, 0x0421, "Nikon D90", ptp_flag_lv, 4288, 2848, 5.50,,
+Nikon, 0x0422, "Nikon D700", ptp_flag_lv, 4256, 2832, 8.45,,
+Nikon, 0x0423, "Nikon D5000", ptp_flag_lv, 4288, 2848, 5.50,,
+Nikon, 0x0424, "Nikon D3000", 0, 3872, 2592, 6.09,,
+Nikon, 0x0425, "Nikon D300S", ptp_flag_lv, 4288, 2848, 5.50,,
+Nikon, 0x0426, "Nikon D3S", ptp_flag_lv, 4256, 2832, 8.45,,
+Nikon, 0x0427, "Nikon D3100", ptp_flag_lv, 4608, 3072, 5.0,,
+Nikon, 0x0428, "Nikon D7000", ptp_flag_lv, 4928, 3264, 4.78,,
+Nikon, 0x0429, "Nikon D5100", ptp_flag_lv, 4928, 3264, 4.78,,
+Nikon, 0x042a, "Nikon D800", ptp_flag_lv, 7360, 4912, 4.88,,
+Nikon, 0x042b, "Nikon D4", ptp_flag_lv, 4928, 3280, 7.30,,
+Nikon, 0x042c, "Nikon D3200", ptp_flag_lv, 6016, 4000, 3.92,,
+Nikon, 0x042d, "Nikon D600", ptp_flag_lv, 6016, 4016, 5.95,,
+Nikon, 0x042e, "Nikon D800E", ptp_flag_lv, 7360, 4912, 4.88,,
+Nikon, 0x042f, "Nikon D5200", ptp_flag_lv, 6000, 4000, 3.92,,
+Nikon, 0x0430, "Nikon D7100", ptp_flag_lv, 6000, 4000, 3.92,,
+Nikon, 0x0431, "Nikon D5300", ptp_flag_lv, 6000, 4000, 3.92,,
+Nikon, 0x0432, "Nikon Df", ptp_flag_lv, 4928, 3280, 7.3,,
+Nikon, 0x0433, "Nikon D3300", ptp_flag_lv, 6000, 4000, 3.92,,
+Nikon, 0x0434, "Nikon D610", ptp_flag_lv, 6016, 4016, 5.97,,
+Nikon, 0x0435, "Nikon D4S", ptp_flag_lv, 4928, 3280, 7.30,,
+Nikon, 0x0436, "Nikon D810", ptp_flag_lv, 7360, 4912, 4.88,,
+Nikon, 0x0437, "Nikon D750", ptp_flag_lv, 6016, 4016, 5.97,,
+Nikon, 0x0438, "Nikon D5500", ptp_flag_lv, 6000, 4000, 3.92,,
+Nikon, 0x0439, "Nikon D7200", ptp_flag_lv, 6000, 4000, 3.92,,
+Nikon, 0x043a, "Nikon D5", ptp_flag_lv, 5568, 3712, 6.40,,
+Nikon, 0x043b, "Nikon D810A", ptp_flag_lv, 7360, 4912, 4.88,,
+Nikon, 0x043c, "Nikon D500", ptp_flag_lv, 5568, 3712, 4.23,,
+Nikon, 0x043d, "Nikon D3400", ptp_flag_lv, 6000, 4000, 3.92,,
+Nikon, 0x043f, "Nikon D5600", ptp_flag_lv, 6000, 4000, 3.92,,
+Nikon, 0x0440, "Nikon D7500", ptp_flag_lv, 5568, 3712, 4.22,,
+Nikon, 0x0441, "Nikon D850", ptp_flag_lv, 8256, 5504, 4.34,,
+Nikon, 0x0442, "Nikon Z 7", ptp_flag_lv, 8256, 5504, 4.34,,
+Nikon, 0x0443, "Nikon Z 6", ptp_flag_lv, 6048, 4024, 5.9,,
+Nikon, 0x0445, "Nikon D3500", ptp_flag_lv, 6000, 4000, 3.92,,
+Sony, 0x079b, "Sony Alpha A68", ptp_flag_lv, 6000, 4000, 3.92,,
+Sony, 0x079c, "Sony Alpha A6300", ptp_flag_lv, 6000, 4000, 3.92,,
+Sony, 0x079e, "Sony Alpha A99II", ptp_flag_lv, 7952, 5304, 4.51,,
+Sony, 0x07a4, "Sony Alpha A6500", ptp_flag_lv, 6000, 4000, 3.92,,
+Sony, 0x07c6, "Sony Alpha A5000", 0, 5456, 3632, 4.25,,
+Sony, 0x094c, "Sony Alpha A7", ptp_flag_lv, 6000, 4000, 5.98,,
```

Comment:
I found some strange at https://github.com/indigo-astronomy/indigo/blob/e63f6e8dee0ce2dc11b7434bac136ab62bfbfbdf/indigo_drivers/ccd_ptp/indigo_ccd_ptp.c#L155

```
	{ SONY_VID, 0x094c, "Sony Alpha A7", 6000, 4000, 5.98 },
	{ SONY_VID, 0x094d, "Sony Alpha A7R", 6000, 4000, 5.98 },
	{ SONY_VID, 0x094e, "Sony Alpha A6000", 6000, 4000, 3.92 },
```

Probably `flags` value is missing.
Set `ptp_flag_lv` for the time being.